global.json

```diff
@@ -1,23 +1,23 @@
 {
-  "tools": {
-    "dotnet": "6.0.100",
-    "runtimes": {
-      "dotnet": [
-        "2.1.7",
-        "$(VSRedistCommonNetCoreSharedFrameworkx6460PackageVersion)"
-      ]
+    "tools": {
+        "dotnet": "7.0.100-alpha.1.21620.6",
```

Comment:
go with `7.0.100-alpha.1.22053.1` . an updated one available and winforms adapted this. So we can be in sync.